lib/plugins/orionUrls.js

```diff
@@ -29,68 +29,86 @@
+    ['POST', /^\/(ngsi9|v1\/registry)\/contextentitytypes\/.+\/attributeDomains\/.+/, 'register'],
+    ['GET', /^\/(ngsi9|v1\/registry)\/contextentitytypes\/.+/, 'discover'],
+    ['POST', /^\/(ngsi9|v1\/registry)\/contextentitytypes\/.+/, 'register'],
+    ['POST', /^\/(ngsi9|v1\/registry)\/contextavailabilitysubscriptions$/, 'subscribe-availability'],
+    ['PUT', /^\/(ngsi9|v1\/registry)\/contextavailabilitysubscriptions\/.+/, 'subscribe-availability'],
+    ['DELETE', /^\/(ngsi9|v1\/registry)\/contextavailabilitysubscriptions\/.+/, 'subscribe-availability'],
+    /* "Classic" NGSI10 operations */
+    ['GET', /^\/(ngsi10|v1)\/contextentities\/.+\/attributes$/, 'N/A'],
+    ['PUT', /^\/(ngsi10|v1)\/contextentities\/.+\/attributes$/, 'N/A'],
+    ['POST', /^\/(ngsi10|v1)\/contextentities\/.+\/attributes$/, 'N/A'],
+    ['DELETE', /^\/(ngsi10|v1)\/contextentities\/.+\/attributes$/, 'N/A'],
+    ['GET', /^\/(ngsi10|v1)\/contextentities\/.+\/attributes\/.+/, 'read'],
+    ['POST', /^\/(ngsi10|v1)\/contextentities\/.+\/attributes\/.+/, 'create'],
+    ['PUT', /^\/(ngsi10|v1)\/contextentities\/.+\/attributes\/.+/, 'update'],
+    ['DELETE', /^\/(ngsi10|v1)\/contextentities\/.+\/attributes\/.+/, 'delete'],
+    ['GET', /^\/(ngsi10|v1)\/contextentities\/.+\/attributes\/.+\/.+/, 'read'],
+    ['PUT', /^\/(ngsi10|v1)\/contextentities\/.+\/attributes\/.+\/.+/, 'update'],
+    ['DELETE', /^\/(ngsi10|v1)\/contextentities\/.+\/attributes\/.+\/.+/, 'delete'],
+    ['GET', /^\/(ngsi10|v1)\/contextentities\/.+\/attributeDomains\/.+/, 'read'],
+    ['GET', /^\/(ngsi10|v1)\/contextentities\/.+/, 'read'],
+    ['PUT', /^\/(ngsi10|v1)\/contextentities\/.+/, 'update'],
+    ['POST', /^\/(ngsi10|v1)\/contextentities\/.+/, 'create'],
+    ['DELETE', /^\/(ngsi10|v1)\/contextentities\/.+/, 'delete'],
+    ['GET', /^\/(ngsi10|v1)\/contextentitytypes\/.+\/attributes$/, 'N/A'],
+    ['GET', /^\/(ngsi10|v1)\/contextentitytypes\/.+\/attributes\/.+/, 'read'],
+    ['GET', /^\/(ngsi10|v1)\/contextentitytypes\/.+/, 'read'],
+    ['GET', /^\/(ngsi10|v1)\/contextentitytypes\/.+\/attributeDomains\/.+/, 'read'],
+    ['POST', /^\/(ngsi10|v1)\/contextsubscriptions$/, 'subscribe'],
+    ['PUT', /^\/(ngsi10|v1)\/contextsubscriptions\/.+/, 'subscribe'],
+    ['DELETE', /^\/(ngsi10|v1)\/contextsubscriptions\/.+/, 'subscribe'],
+    /* New operations in v1/ (note that they don't use the "ngsi10" alternative in the pattern) */
+    ['GET', /^\/v1\/contextentities/, 'read'],
+    ['POST', /^\/v1\/contextentities/, 'create'],
+    ['GET', /^\/v1\/contextsubscriptions/, 'read'],
+    ['GET', /^\/v1\/contextsubscriptions\/.+/, 'read'],
+    ['GET', /^\/v1\/contexttypes/, 'read'],
+    ['GET', /^\/v1\/contexttypes\/.+/, 'read'],
+
+    /* V2 Operations */
+    ['GET', /^\/v2$/, 'read'],
+    ['GET', /^\/v2\/entities$/, 'read'],
+    ['GET', /^\/v2\/entities\/.+/, 'read'],
+    ['POST', /^\/v2\/entities$/, 'create', {op: 'append'}],
```

Comment:
Not sure if at the end NGSIv2 uses `op=append` or `options=...,append,...`. I'm checking it...